Source/Blazorise.Bootstrap/Styles/_link.scss

```diff
@@ -0,0 +1,4 @@
+.link-unstyled {
+    text-decoration: none;
+    color: inherit !important;
```

Comment:
Prefer to have styles without `!important`. That way users can overwrite their own styles. Do the same on all other providers.

PS. you can have higher specificity with `a.link-unstyled`.